guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -12,19 +12,21 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
 [options="nowrap" subs="+quotes,attributes"]
 ----
 {installer-scenario} \
-  --foreman-db-host _postgres.example.com_ \
-  --foreman-db-password _Foreman_Password_ \
-  --foreman-db-database foreman \
-  --foreman-db-manage false \
+  --katello-candlepin-manage-db false \
   --katello-candlepin-db-host _postgres.example.com_ \
   --katello-candlepin-db-name candlepin \
+  --katello-candlepin-db-user candlepin \
   --katello-candlepin-db-password _Candlepin_Password_ \
-  --katello-candlepin-manage-db false \
   --foreman-proxy-content-pulpcore-manage-postgresql false \
   --foreman-proxy-content-pulpcore-postgresql-host _postgres.example.com_ \
   --foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
-  --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_
-  --foreman-proxy-content-pulpcore-postgresql-user pulp
+  --foreman-proxy-content-pulpcore-postgresql-user pulp \
+  --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_ \
```

Comment:
suggestion: "_My_Pulpcore_Password_"

same for Foreman & in guides/common/modules/proc_migrating-to-external-databases.adoc

Reply:
Can we leave that for another PR?

Reply:
sure!